config/environments/preprod.rb

```diff
@@ -5,4 +5,6 @@
   # Override any production defaults here
   config.x.git_api_endpoint = \
     "https://get-into-teaching-api-dev.london.cloudapps.digital/api"
+  config.x.google_maps_key = ENV["GOOGLE_MAPS_KEY"].presence || \
```

Comment:
This is unneeded since its getting set via the inheritance from the `production.rb`